manifests/operator-service-account-rbac.yaml

```diff
@@ -36,19 +36,19 @@ rules:
   - list
   - watch
 # all verbs allowed for event streams (Zalando-internal feature)
-# - apiGroups:
-#   - zalando.org
-#   resources:
-#   - fabriceventstreams
-#   verbs:
-#   - create
-#   - delete
-#   - deletecollection
-#   - get
-#   - list
-#   - patch
-#   - update
-#   - watch
+- apiGroups:
+  - zalando.org
+  resources:
+  - fabriceventstreams
+  verbs:
+  - create
+  - delete
+  - deletecollection
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
I would not remove comments from this example manifest because it is probably used by many when first testing our operator. If the fes CRD is not present this can produce errors. Maybe we can update the cluster role as part of the e2e test.